utilities_common/chassis.py

```diff
@@ -16,3 +16,32 @@ def get_chassis_local_interfaces():
                 lst = data[1].split(",")
                 return lst
     return lst
+
+
+def is_smartswitch():
+    return hasattr(device_info, 'is_smartswitch') and device_info.is_smartswitch()
+
+
+# utility to get dpu module name list
+def get_all_dpus():
```

Comment:
What does this return if get_dpu_list() is not defined?

Reply:
@vvolam Expects this function to return an empty list in any failure case.